Packs/HelloWorldPremium/ReleaseNotes/1_0_7.md

```diff
@@ -0,0 +1,4 @@
+
+#### Classifiers
+##### HelloWorldPremium
+- Change classifier-HelloWorldPremium_5_9_9 ID.
```

Comment:
you can change to the generic maintenance message.

Reply:
done